src/views/TasksDetailsView.vue

```diff
@@ -116,58 +140,46 @@ window.scrollTo({ top: 0, behavior: "smooth" });
+      </v-card>
+      <div class="pointsTemplate__container ml-4">
+        <v-sheet class="pointsTemplate__top"
+          :style="{ height: pointsTemplateHeight + 'px', backgroundColor: backgroundColor }">
+          <hub-tooltip :tooltipText="$t('taskDetails.pointsTooltip')">
+            <template #default>
+              <span class="pointsNumber">{{ taskDetails.points }} <span v-if="showPointsText"
+                  style="font-size: 24px;">pkt</span>
+              </span>
+            </template> </hub-tooltip>
+        </v-sheet>
+        <div class="pointsTemplate__bottom" :style="{ backgroundColor: backgroundColor }"></div>
+      </div>
+    </div>
     <v-card class="my-4 px-2 py-2">
       <span class="px-2 text-subtitle-1">{{
         $t("taskDetails.additionalInfo")
       }}</span>
       <v-card-text>{{ taskDetails.additionalDescription }}</v-card-text>
     </v-card>
-    <v-card
-      v-for="(image, attachment) of additionalImages"
-      :key="attachment"
-      class="px-2 py-2 my-2"
-    >
-      <span class="px-2 text-subtitle-2"
-        >{{ $t("taskDetails.attachment") }} {{ attachment + 1 }}</span
-      >
+    <v-card v-for="(image, attachment) of additionalImages" :key="attachment" class="px-2 py-2 my-2">
+      <span class="px-2 text-subtitle-2">{{ $t("taskDetails.attachment") }} {{ attachment + 1 }}</span>
       <img class="h-100 w-100 pt-2" :src="image" />
     </v-card>
-    <user-mini-card
-      :user="createrDetails"
-      :date="taskDetails.creationDate"
-      dateText="taskDetails.taskCreationDate"
-    />
-    <user-mini-card
-      v-if="taskDetails.acceptDate"
-      :user="accepterDetails"
-      :date="taskDetails.acceptDate"
-      dateText="taskDetails.taskAcceptedDate"
-      reverse
-    />
+    <user-mini-card :user="createrDetails" :date="taskDetails.creationDate" dateText="taskDetails.taskCreationDate" />
+    <user-mini-card v-if="taskDetails.acceptDate" :user="accepterDetails" :date="taskDetails.acceptDate"
+      dateText="taskDetails.taskAcceptedDate" reverse />
     <v-card class="my-4 px-2 py-2">
       <span class="px-2 text-subtitle-1">{{
         $t("taskDetails.completedByUsers")
       }}</span>
       <div class="d-flex flex-wrap justify-center px-8 pt-4 ga-2">
-        <btn-with-dropdown
-          v-for="team of teams"
-          :key="team.teamId"
-          :title="team.name"
-          :color="team.color"
-          :icon="team.logo"
-          :items="['Kacper', 'Laura', 'Krzysiek']"
-        ></btn-with-dropdown>
+        <btn-with-dropdown v-for="team of teams" :key="team.teamId" :title="team.name" :color="team.color"
+          :icon="team.logo" :items="['Kacper', 'Laura', 'Krzysiek']" />
       </div>
-      <apexchart
-        class="py-4"
-        type="pie"
-        :options="chartOptions"
-        :series="series"
-      ></apexchart>
+      <apexchart class="py-4" type="pie" :options="chartOptions" :series="series"></apexchart>
```

Comment:
```suggestion
      <apexchart class="py-4" type="pie" :options="chartOptions" :series="series"/>
```
Jak masz puste `></>` to warto to zastąpic tym `/>`, zerknij czy w innych miejsach są takie sytuacje i tez prosze podmien :D